packages/fx-core/src/plugins/resource/appstudio/plugin.ts

```diff
@@ -328,27 +329,27 @@ export class AppStudioPluginImpl {
 
   public createManifestForRemote(
     ctx: PluginContext,
-    maybeSelectedPlugins: Result<Plugin[], FxError>,
+    // maybeSelectedPlugins: Result<Plugin[], FxError>,
     manifest: TeamsAppManifest
   ): Result<[IAppDefinition, TeamsAppManifest], FxError> {
-    if (maybeSelectedPlugins.isErr()) {
-      return err(maybeSelectedPlugins.error);
-    }
-    const selectedPlugins = maybeSelectedPlugins.value;
-    if (selectedPlugins.some((plugin) => plugin.name === "fx-resource-bot")) {
-      const capabilities = (ctx.projectSettings?.solutionSettings as AzureSolutionSettings)
-        .capabilities;
-      const hasBot = capabilities?.includes(BotOptionItem.id);
-      const hasMsgExt = capabilities?.includes(MessageExtensionItem.id);
-      if (!hasBot && !hasMsgExt) {
-        return err(
-          AppStudioResultFactory.SystemError(
-            AppStudioError.InternalError.name,
-            AppStudioError.InternalError.message
-          )
-        );
-      }
+    // if (maybeSelectedPlugins.isErr()) {
+    //   return err(maybeSelectedPlugins.error);
+    // }
+    // const selectedPlugins = maybeSelectedPlugins.value;
+    // if (selectedPlugins.some((plugin) => plugin.name === "fx-resource-bot")) {
+    const capabilities = (ctx.projectSettings?.solutionSettings as AzureSolutionSettings)
+      .capabilities;
+    const hasBot = capabilities?.includes(BotOptionItem.id);
+    const hasMsgExt = capabilities?.includes(MessageExtensionItem.id);
+    if (!hasBot && !hasMsgExt) {
```

Comment:
The original logic throw error when bot plugin is selected without bot/me capability.

But here, it throws error when the app is not bot/me app.